tools/chainbuilder/README.md

```diff
@@ -0,0 +1,27 @@
+# Chainbuilder
+
+`chainbuilder` is a tool for building a Celestia chain for testing and development purposes.
+
+## Usage
+
+Use `go` to run the binary as follows:
+
+```
+go run ./tools/chainbuilder
+```
+
+This will create a directory with the name `testnode-{chainID}`. All files will be populated and blocks generated based on specified input. You can run a validator on the file system afterwards by calling:
+
+```
+celestia-appd start --home /path/to/testnode-{chainID}
```

Comment:
[nit] It might be helpful to mention how the pruning option may need adjustment. When I ran this and pointed it to the generated directory by chainbuilder, I encountered this error:

```
celestia-appd start --home ./testnode-1YgtBH --force-no-bbr
Error: cannot enable state sync snapshots with 'everything' pruning setting: error in app.toml
```

I had to modify the `pruning` setting in `app.toml` to resolve the error. Up to you, but it might be worth mentioning in the documentation.